encoding/video-artifacts.md

```diff
@@ -261,6 +261,17 @@ it's better to do this under 16bit to avoid precision problems.
 
 ![](images/overflow_notice.jpg)
 
+Here's some examples if you want to use `std.Expr()`.
```

Comment:
Good of you to add these. I thought whether we should provide them or have the user figure it out by themselves, but I believe at this point it's better to go with an example to prevent avoidable errors.

Reply:
I think it's hard to let the user figure it out without having a brief introduction about the conversion between limited range and full range. Or we should make another page introducing these mathematic stuff about YUV, RGB  and range.